prow/jobs.yaml

```diff
@@ -38,10 +38,9 @@ kubernetes/heapster:
 
 kubernetes/kubernetes:
 - name: pull-kubernetes-bazel
-  always_run: true
-  context: Jenkins Bazel Build
+  #  context: Jenkins Bazel Build
```

Comment:
did you mean to comment this out? shouldn't turning off "always_run" and changing the trigger be sufficient?